chore: align README for non-prefixed packages (#2994)
vaadin/web-components: packages/menu-bar/README.md

# @vaadin/menu-bar

> A web component used to create a horizontal button bar with hierarchical drop-down menus.

Reviewer: ```suggestion
A web component for creating a horizontal button bar with hierarchical drop-down menus.
```

Author: Done.
